core-services/release-controller/deploy-ocp-controller-ppc64le.yaml

```diff
@@ -1,6 +1,57 @@
 kind: List
 apiVersion: v1
 items:
+# Bootstrap the environment for the amd64 tests image
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    name: release-controller-bootstrap
+    namespace: ocp-ppc64le
+    annotations:
+      release.openshift.io/config: |-
```

Comment:
why did you create this?  Image streams without this annotation are ignored

Reply:
The release controller only considers  imagestreams with the release config annotation.  Without it, it wasn't picking up the stream that I wanted it to.  It would always simply find the first "tests" stream and use it.

Reply:
I have verified that the tools-image-stream-tag parameter negates that need for a release-config annotation.  I have removed it.